docs/API.md

```diff
@@ -1580,24 +1580,16 @@ for (Result<DeleteError> result : results) {
 ```
 
  <a name="selectObjectContent"></a>
-### selectObjectContent(String bucketName, String objectName, String sqlExpression, InputSerialization is, OutputSerialization os, boolean requestProgress, Long scanStartRange, Long scanEndRange, ServerSideEncryptionCustomerKey ssec)
-`public SelectResponseStream selectObjectContent(String bucketName, String objectName, String sqlExpression, InputSerialization is, OutputSerialization os, boolean requestProgress, Long scanStartRange, Long scanEndRange, ServerSideEncryptionCustomerKey ssec)` _[[Javadoc]](http://minio.github.io/minio-java/io/minio/MinioClient.html#selectObjectContent-java.lang.String-java.lang.String-java.lang.String-io.minio.messages.InputSerialization-io.minio.messages.OutputSerialization-boolean-java.lang.Long-java.lang.Long-io.minio.ServerSideEncryptionCustomerKey-)_
+### selectObjectContent(SelectObjectContentArgs args)
+`public SelectResponseStream selectObjectContent(SelectObjectContentArgs args)` _[[Javadoc]](http://minio.github.io/minio-java/io/minio/MinioClient.html#selectObjectContent-io.minio.SelectObjectContentArgs-)_
 
 Selects content of a object by SQL expression.
 
 __Parameters__
 
 | Parameter           | Type                                | Description                           |
 |:--------------------|:------------------------------------|:--------------------------------------|
-| ``bucketName``      | _String_                            | Name of the bucket.                   |
-| ``objectName``      | _String_                            | Object name in the bucket.            |
-| ``sqlExpression``   | _String_                            | SQL expression.                       |
-| ``is``              | _[InputSerialization]_              | Input specification of object data.   |
-| ``os``              | _[OutputSerialization]_             | Output specification of result.       |
-| ``requestProgress`` | _boolean_                           | Flag to request progress information. |
-| ``scanStartRange``  | _Long_                              | scan start range of the object.       |
-| ``scanEndRange``    | _Long_                              | scan end range of the object.         |
-| ``sse``             | _[ServerSideEncryptionCustomerKey]_ | SSE-C type server-side encryption.    |
+| ``args``            | _SelectObjectContentArgs_           | Arguments.                            |
```

Comment:
`SelectObjectContentArgs` requires `[]` around to make it as link.

Reply:
My bad, thanks!